docs/source/concepts.rst

```diff
@@ -214,6 +214,53 @@ on an H2O model that requires the h2o conda packages:
     One caveat with Conda Packages here, is that it does not work with AWS Lambda 
     deployment due to the limitation of the AWS Lambda platform.
 
+Using other Docker base images
+^^^^^^^^^^^^^^^^
+
+By default, BentoML uses a default Docker base image and installs your model and its dependencies on top of it. This base image contains all of BentoML's dependencies and an installation of `conda` which helps BentoML to package and use the right Python version for your bundle.
+
+However, there may be times when you need to use other Docker images (e.g. have some pre-build dependencies layers, company base image, using an Alpine-based image, etc.). BentoML makes it really easy to switch between base images by doing allowing you to specify a `docker_base_image`.
+
+.. code-block:: python
+
+  # e.g. using BentoML Alpine-based images
+  @env(docker_base_image="bentoml/model-server:0.8.12-alpine3.7")
+  @artifacts([SklearnModelArtifact('model')])
+  class ExamplePredictionService(BentoService):
+    ...
+
+In fact, one such base image that many may find useful are the Alpine-based images. The original base image weighs in at roughly `~853MB` whereas the Alpine version weighs in at `~360MB`. 
+
+.. code-block:: bash
+
+  > docker image ls
+
+  REPOSITORY                               TAG                 IMAGE ID            CREATED              SIZE
+  bentoml/model-server                     0.8.12-alpine       109b451ed537        6 minutes ago        360MB
+  bentoml/model-server                     0.8.12              f034fa23264c        33 minutes ago       853MB
+
+This means that each image built on top of Alpine based images will be significantly smaller. 
+
+.. code-block:: bash
+
+  > docker image ls
+
+  REPOSITORY                               TAG                 IMAGE ID            CREATED              SIZE
+  jzhao2k19/iris                           latest              bfc9b81c7535        About a minute ago   1.54GB
+  jzhao2k19/iris-alpine                    latest              4e8d87a0c18a        4 minutes ago        577MB
+
+However, as with using any alternative Docker base image, there are a few things to keep in mind. The regular base image uses `conda`, whereas the Alpine image does not. This has a few consequences. BentoML uses `conda` to ensure the Python version used matches the one you used to save your bundle. Additionally, this means that BentoML will ignore the `environment.yml`, meaning that user-defined `conda` packages and dependencies will be ignored. In the following example, only `pandas` will be installed, as the `conda_channels` and `conda_dependencies` will be ignored.
+
+.. code-block:: python
+
+  @bentoml.env(
+    pip_dependencies=['pandas'],
+    conda_channels=['h2oai'],
+    conda_dependencies=['h2o==3.24.0.2'],
+    docker_base_image="bentoml/model-server:0.8.12-alpine3.7"
```

Comment:
could you make sure this base image here in the example matches the name we have in the docker release script? `bentoml/model-server:0.8.12-alpine3.7`?

I think you may also want to rename the docker file here to `Dockerfile-alpine-py36`, `Dockerfile-alpine-py37`.. and add them to the `docker/model-server/release.sh` script.

We currently run that release script after every PyPI release, which pushes the new image to docker hub under, e.g. `bentoml/model-server:0.8.2-alpine-py36`.